dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -91,6 +91,12 @@ const listItemStyles = (palette: Palette) => css`
 const textStyles = (palette: Palette) => css`
 	${textSans.small({ fontWeight: 'regular', lineHeight: 'regular' })};
 	color: ${palette.text.keyEvent};
+
+	&:hover {
```

Comment:
this is very cool! just to be pedantic, I think we should underline when we hover over any part of the link (e.g. when we hover over the timestamp or the dot)

Reply:
but just to be clear, I don't think we should underline the timestamp, just that the text should become underlined when we hover over the timestamp - I think that's the correct behaviour?

Reply:
@alinaboghiu gotcha, I have changed this now to include hover over the whole card. Please let me know what you think.